### PR TITLE
added sub-08 to ds008/participants.tsv

### LINKED ISSUE #14: The participants.tsv in ds008 doesn't have sub-08 listed.
Is [this](https://github.com/INCF/BIDS-examples/blob/master/ds008/participants.tsv#L9) an error or is it allowed to not list all of the subjects in the participants.tsv file?

### PR DESCRIPTION
**Pull request by suyashdb**
Closes #14 .

### CODE REVIEW

File: ds008/participants.tsv [hunk 1]
```diff
@@ -6,6 +6,7 @@ sub-04	M	31
 sub-05	M	22
 sub-06	F	21
 sub-07	M	24
+sub-08  M 26
```
Comment (chrisfilo) [inline ds008/participants.tsv]: are those tabs?

**Comment by suyashdb** [comment]
Can be closed.

**Comment by chrisfilo** [comment]
Did you mean merged?

**Comment by suyashdb** [comment]
Oh yes can be merged!

**Pull request merged**